domains/manulucenaportfolio.is-a.dev.json.json

```
@@ -0,0 +1,10 @@
+{
+    "subdomain": "manulucenaportfolio",
+    "owner": {
+      "email": "mlucenasalas@gmail.com"

```

username is required 
```suggestion
      "username": "manulucena12",
      "email": "mlucenasalas@gmail.com"
```